addons/mail/static/src/models/message_view.js

```diff
@@ -230,6 +233,20 @@ registerModel({
             }
             return clear();
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeFailureNotificationIconClassName() {
+            return clear();
+        },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeFailureNotificationIconLabel() {
+            return clear();
```

Comment:
logically speaking, I feel this strange to use `clear()` to call the default which has a meaningful value. `clear()` logically should be something like empty or nothing, isn't it? maybe it's better to just return the value inside the compute?

Reply:
This is just because the need to be override on other module. The pattern is strange but I don't really think there is an other option.

Reply:
I get your point, but also it makes sense for default to have a meaningful, and it's not a good idea to set a non-meaningful value.

I think there are improvements to do with default in the future. But also choosing a default feels in a way arbitrary, so here relying on default or returning an explicit compute value are both valid choices.